CDCgov/prime-reportstream
# update schema for anavasidx-covid-19

This PR ...

**updates the default value for processing_mode_code to "P" for schema anavasidx-covid-19 and updates schema to use testing lab demographic information for ordering facility demographic information if not available.**

Test Steps:
1. create org anavasidx locally by copy/pasting below into organizations.yml

\- name: anavasidx
    description: anavasidx
    jurisdiction: FEDERAL
    senders:
     \- name: default
         organizationName: anavasidx
         topic: covid-19
         customerStatus: active
         schemaName: direct/anavasidx-covid-19
         format: CSV

2. run ./prime data ---input-client anavasidx --input <test-file-location> --output-dir <location-to-output-test-file> --route locally using attached CSV test file as input

[Anavasi-test-data.csv](https://github.com/CDCgov/prime-reportstream/files/11236637/Anavasi-test-data.csv)

4. Verify that output file is created with MSH-11 (processing_mode_code) value of "P" and that ORC-21, ORC-22 and ORC-23 fields are populated"

## Changes
- updates the default value for processing_mode_code to "P" for schema anavasidx-covid-19.
- updates schema to use testing lab demographic information for ordering facility demographic information if not available.

### Testing
- [x ] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
